frontend/src/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
           <Route path="/home" element={<Dashboard />} />
           <Route path="/profile" element={<Profile />} />
           <Route path="/profile/edit" element={<EditProfile />} />
+          <Route path="/profile/follows" element={<Follows />} />
```

Comment:
1. what is the difference with /profile and /profile/follows?
2. if i am visiting someones profile and I am going to refresh, how can you handle to keep the certain profile/page?
Make your route ready and I suggest:
`<Route path="/profile/:id" element={<Follows />} />`